source/grid/grid_tools.cc

```diff
@@ -2662,7 +2662,7 @@ namespace GridTools
     static inline std::bitset<3> lookup (const MATCH_T &)
     {
       // The 1D case is trivial
-      return 4; // [true ,false,false]
+      return 1; // [true ,false,false]
```

Comment:
The question is whether the comment is supposed to list the bits in their order (i.e., indexed as `bitset` wants them) or if you intend them to be digit quantifiers (like write numbers, 124=1*100+2*10+4*1). 

In other words, the confusion happened because you assumed a particular convention in stating the comment. That convention is still implicit -- might be best to make this clear in the comment, e.g.
```
  return 1; // bits[0]=true, bits[1]=false, bits[2]=false;
```